### ISSUE TITLE
Feature Request: Instant Upload

### ISSUE THREAD

**Issue by dshanske** [swentel/indigenous-android#404]
How about an instant upload function, like some Android apps have, that will upload all photos you take to your media endpoint?

**Comment by swentel** [comment]
So from a configuration point of a view, you'd enable this in settings and then configure one or more folders to keep an eye on where those photos are saved and then send?

I think there's an event send out in Android when a photo is saved, so it shouldn't be that hard I think, but I'd have to check.

**Comment by dshanske** [comment]
@swentel Basically yes. The paired feature would be using q=source on the media endpoint to pick already uploaded photos.

**Comment by dshanske** [comment]
Similar functionality is in the Dropbox, Nextcloud, Amazon Photos, etc

**Comment by swentel** [comment]
Right, using the nextcloud feature as well. Not 100% sure whether I'll implement this as it's tricky stuff, but especially since I won't use this feature myself.
